controllers/datadogagent/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "go.inferGopath": false
+}
```

Comment:
this file should not be committed

Reply:
[5d5d7c2](https://github.com/DataDog/datadog-operator/pull/679/commits/5d5d7c29c89e893d42873a97a90b181296c2475a)